djstripe/stripe_objects.py

```diff
@@ -968,21 +1014,22 @@ def get_default_account(cls):
 
 class StripeCard(StripeSource):
     """
-You can store multiple cards on a customer in order to charge the customer later.
-(Source: https://stripe.com/docs/api/python#cards)
+    You can store multiple cards on a customer in order to charge the customer later.
```

Comment:
add "A model which represents a Stripe Card object." to the beginning of this like the other models.

Also, I'm fairly sure it should be "a model that", but I'm not sure. Can you look it up and post a link?